intellij-ext/build.gradle.kts

```diff
@@ -89,9 +89,9 @@ tasks {
                 into(pluginName.get())
             }
             // NOTE: comment this line when developing locally without either of shared libs
-            if (!System.getenv("CI_BUILD_PLUGIN").toBoolean()) {
-                if (!inputs.sourceFiles.files.contains(File(lib))) throw StopExecutionException("Expected library: $lib >> Not Found")
-            }
+//            if (!System.getenv("CI_BUILD_PLUGIN").toBoolean()) {
```

Comment:
Is this intentional?

Reply:
> Do we know why this only happened on CLion?

Not sure, from the question i asked in the forum they said all UI code must run in a specific thread. Ill test with @t4lz to confirm if the issue is reproduced